### PR TITLE
Twilio sample for app engine flex.

### PR DESCRIPTION
**Pull request by SurferJeffAtGoogle**
Twilio sample for app engine flex.

### CODE REVIEW

File: appengine/flexible/twilio/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+```
+
+## Run locally
+
+you can run locally using PHP's built-in web server:
+
+```sh
+export TWILIO_ACCOUNT_SID=your-account-sid
+export TWILIO_AUTH_TOKEN=your-auth-token
+export TWILIO_NUMBER=your-twilio-number
```
Comment (tmatsuo) [inline appengine/flexible/twilio/README.md]: These secrets, now you need to specify them twice, once in the `app.yaml` and once in the terminal. I think we can converge them by having a configuration file template for them like our getting started app.

How about to have `settings.yml.dist` file in the repo which has the placeholders for those secrets and have the app always read from it?

**Comment by tmatsuo** [comment]
LGTM with one minor suggestion!

File: appengine/flexible/twilio/test/DeployAppEngineFlexTest.php [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+use Google\Cloud\TestUtils\FileUtil;
+use Symfony\Component\Yaml\Yaml;
+
+class DeployAppEngineFlexTest extends \PHPUnit_Framework_TestCase
+{
+    use AppEngineDeploymentTrait;
+
+    public function beforeDeploy()
+    {
+        $tmpDir = FileUtil::cloneDirectoryIntoTmp(__DIR__ . '/..');
```
Comment (bshaffer) [inline appengine/flexible/twilio/test/DeployAppEngineFlexTest.php]: We should move this class into `google/cloud-tools` at some point

**Comment by bshaffer** [comment]
LGTM!

**Pull request merged**